### DISCUSSION TITLE
Getting issue with mvn clean package for Dbeaver Develop in Eclipse documentation project

### DISCUSSION THREAD

**Discussion by RahulSindhu007** [dbeaver/dbeaver#34896] [DBeaver Development] [answered]
Followed https://dbeaver.com/docs/dbeaver/Develop-in-Eclipse/ and able to run dbeaver in eclipse but getting issue while mvn clean package.

Getting this:
Failed to load p2 repository with ID 'local-contrib' from location https://p2.dev.dbeaver.com/eclipse-repo/24.1.1: Unable to read repository at file:/C:/Users/rahul/.m2/repository/.cache/tycho/https/p2.dev.dbeaver.com/eclipse-repo/.idx.
I am new to this need help.

**Comment by TB-HunBoka** [comment, marked as answer]
https://p2.dev.dbeaver.com/eclipse-repo/24.1.1 doesn't exists currently. The 24.1.2 and 24.1.3 and newest 24.1.4 exists. Probably you are working with an outdated version, please pull the latest git version of the dbeaver and dbeaver-common
